test/e2e_node/remote/utils.go

```diff
@@ -21,16 +21,15 @@ import (
 	"path/filepath"
 
 	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/test/e2e_node/builder"
```

Comment:
nit: group local repo imports in their own group

Reply:
Will regroup.

Reply:
this is also non-blocking but hopefully can go with other changes

Reply:
Done.